test/browser_tests/spec_suites/shared/shared_footer.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var Footer = require( '../../shared_objects/footer.js' );
+
+describe( 'The Footer Component', function() {
+  var _sharedObject;
+  var _linkLookupArray = [ 'accessibility',
+                           'foia',
+                           'office of inspector general',
+                           'open government',
+                           'plain writing',
+                           'privacy, policy & legal notices',
+                           'tbd content',
+                           'usa.gov'
+                        ];
+
+  beforeEach( function() {
+    _sharedObject = new Footer();
+    _sharedObject.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+      expect( _sharedObject.footer.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include navList', function() {
+      expect( _sharedObject.navList.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include navLinks and navLinks should be valid', function() {
+    _sharedObject.navLinks.count().then( function( count ) {
+      expect( count > 0 ).toBe( true );
+    } );
+
+    _sharedObject.navLinks.each( function( element ) {
+      element.getText().then( function( text ) {
+        expect( _linkLookupArray.indexOf( text.toLowerCase() ) >= 0 )
```

Comment:
Doesn't have to happen now, but wrong link hrefs may be easier to miss than the display text, so it may make sense to test those too. (see https://github.com/cfpb/cfgov-refresh/pull/845/, for instance.)